### PR TITLE
Adding performance IO workload for Azure

### PR DESCRIPTION
**Pull request by rsevilla87**
Adding a simple FIO workload to check if AWS disk performance is suitable for etcd.
PTAL @sallyom

### CODE REVIEW

File: ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+build_root:
+  image_stream_tag:
+    name: "4.3"
```
Comment (stevekuznetsov) [inline ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml]: Are you really pegged to 4.3?

File: ci-operator/config/cloud-bulldozer/plow/OWNERS [hunk 2]
```diff
@@ -0,0 +1,4 @@
+approvers:
+- rsevilla87
+reviewers:
+- rsevilla87
```
Comment (petr-muller) [inline ci-operator/config/cloud-bulldozer/plow/OWNERS]: Who will review **your** PRs? Consider adding at least one more reviewer.

File: ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    tag: golang-1.12
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: etcd-perf-test-daily
```
Comment (petr-muller) [inline ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml]: You'll need to add also `tag_specification` needed by ci-operator to know what version of OpenShift to install.

File: core-services/prow/02_config/_plugins.yaml [hunk 4]
```diff
@@ -1272,13 +1290,44 @@ lgtm:
...
+  - shrug
+  - sigmention
+  - size
+  - skip
+  - trigger
+  - verify-owners
+  - owners-label
+  - wip
+  - yuks
+  - approve
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml]: You are enabling plugins that handle approvals and reviews but you are not configuring Tide for automated merges (that act on approvals and reviews). Is that intentional?

File: ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml (hunk 3 above)
Comment (petr-muller) [reply ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml]: That's also the reason why the job's rehearsal fails.

File: core-services/prow/02_config/_plugins.yaml [hunk 5]
```diff
@@ -109,6 +109,9 @@ approve:
 - repos:
   - openshift/extended-platform-tests
   require_self_approval: false
+- repos:
+  - cloud-bulldozer/plow
+  require_self_approval: false
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml]: You do not need this if you only want periodics.

File: core-services/prow/02_config/_plugins.yaml [hunk 6]
```diff
@@ -860,6 +863,21 @@ config_updater:
...
+  - endpoint: http://cherrypick
+    events:
+    - issue_comment
+    - pull_request
+    name: cherrypick
+  - endpoint: http://needs-rebase
+    events:
+    - issue_comment
+    - pull_request
+    name: needs-rebase
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml]: You do not need this if you only want periodics.

File: core-services/prow/02_config/_plugins.yaml [hunk 7]
```diff
@@ -1272,13 +1290,44 @@ lgtm:
 - repos:
   - openshift/extended-platform-tests
   review_acts_as_lgtm: true
+- repos:
+  - cloud-bulldozer/plow
+  review_acts_as_lgtm: true
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml]: You do not need this if you only want periodics.

File: core-services/prow/02_config/_plugins.yaml (hunk 4 above)
Comment (petr-muller) [reply core-services/prow/02_config/_plugins.yaml]: You do not need any of this if you only want periodics.

File: core-services/prow/02_config/_plugins.yaml [hunk 8]
```diff
@@ -860,6 +860,16 @@ config_updater:
...
+  cloud-bulldozer/plow:
+  - endpoint: http://refresh
+    events:
+    - issue_comment
+    name: refresh
+  - endpoint: http://cherrypick
+    events:
+    - issue_comment
+    - pull_request
+    name: cherrypick
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml]: You don't need this hunk if you only need periodics now.

File: core-services/prow/02_config/_plugins.yaml [hunk 9]
```diff
@@ -1279,6 +1289,30 @@ owners:
...
+  - lifecycle
+  - override
+  - pony
+  - retitle
+  - shrug
+  - sigmention
+  - size
+  - skip
+  - trigger
+  - verify-owners
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml]: You don't need this hunk if you only need periodics now.

Comment (sallyom) [reply core-services/prow/02_config/_plugins.yaml]: @petr-muller he doesn't need any changes to this plugins file, right?

Comment (petr-muller) [reply core-services/prow/02_config/_plugins.yaml]: @sallyom correct, no plugins should be needed for this use case

Comment (rsevilla87) [reply core-services/prow/02_config/_plugins.yaml]: Ok, I have removed all changes at the _plugins.yaml file

**Comment by petr-muller** [comment]
LGTM

@rsevilla87 feel free to remove @jtaleric to merge this PR, you can add him anytime later once `openshift` org is aware of him.

**Comment by jtaleric** [comment]
:D My request is in! Feel free to remove me thought!

**Pull request merged**